internal/handler/msg_logout.go

```diff
@@ -26,6 +26,7 @@ import (
 // MsgLogout implements `logout` command.
 func (h *Handler) MsgLogout(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg, error) {
 	conninfo.Get(ctx).SetAuth("", "")
+	conninfo.Get(ctx).SetMechanism("")
```

Comment:
This looks fine to me, although I'd almost always rather have the auth and mechanism reset at once on a single mutex with something like `conninfo.Get(ctx).LogoutSCRAM()` or `ResetSCRAM()`.

Reply:
How about if I make mechanism part of Auth so I can do below?
`conninfo.Get(ctx).SetAuth("", "","")`

Reply:
Sounds even better to me.